app/web/components/AppRoute.tsx

```diff
@@ -48,6 +50,14 @@ export const useAppRouteStyles = makeStyles((theme) => ({
       )})`,
     },
   },
+  "@global body": {
+    height: "100%",
```

Comment:
I think we had this a while back (setting 100% height on html and body) but do you happen to remember why we removed it @lucaslcode?

The only thing I remember is the weird background problem where the background colour of body ends at 100%, but this solution doesn't appear to suffer from that so maybe this has always been the way to go 🤔

Reply:
Can't remember!